apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useMemo } from 'react';
+
+export default function PaneContent(paneName: string) {
+  const dimensionOptions = new Set(['size', 'minSize', 'maxSize'])
```

Comment:
We don't need to create this set on every function call => let's move it outside of this function to not recreate it

Reply:
[a91162c](https://github.com/DevExpress/DevExtreme/pull/26972/commits/a91162c87227d06b8645ebae24f59cb7473ae8ad)